src/tls13utils.rs

```diff
@@ -429,8 +443,12 @@ pub(crate) fn encode_u16(bytes: &Bytes) -> Result<Bytes, TLSError> {
     }
 }
 
-/// TLS encode the `bytes` with [`u24`] length.
-pub(crate) fn lbytes3(bytes: &Bytes) -> Result<Bytes, TLSError> {
+/// Attempt to TLS encode the `bytes` with [`u24`] length.
+///
+/// On success, return a new [Bytes] slice such that its first three bytes encode the
+/// big-endian length of `bytes` and the remainder equals `bytes`. Return a [TLSError] if
+/// the length of `bytes` exceeds what can be encoded in three bytes.
+pub(crate) fn encode_lbytes3(bytes: &Bytes) -> Result<Bytes, TLSError> {
```

Comment:
I generally prefer the `uXX` style. But consistency is even more important, so I'm fine with this.

Reply:
If you prefer the `uXX` style, I'm fine with changing it accordingly. Perhaps `check_u16_encoding` is even a bit more descriptive than `check_lbytes2`, for instance.

Reply:
Yeah I think it's more descriptive. But I'm fine either way for this PR.

Reply:
I changed the naming scheme of that family of functions, so it's hopefully more descriptive than before.